app/form_models/publishers/job_listing/about_the_role_form.rb

```diff
@@ -38,9 +44,17 @@ def params_to_save
       safeguarding_information:,
       further_details_provided:,
       further_details:,
+      flexi_working_details_provided:,
     }
   end
 
+  # Overriding load_form to include :flexi_working_details_provided since it's not part of `self.fields`. This is necessary because
+  # :flexi_working_details_provided is explicitly defined as a boolean attribute.
```

Comment:
is there any danger of an error due to this being missing from the list of fields? Do we need the fields method any more?